src/components/addtodo/AddTodo.js

```diff
@@ -7,11 +7,21 @@ export const AddTodo = () => {
     const dispatch = useDispatch()
     let inputRef = useRef('')
 
+    const scrollToBottom = () => {
+        window.scrollTo({
+            top: document.documentElement.scrollHeight+1000,
+            behavior: 'smooth',
+        });
+    };
+    
     const addHandler = () => {
-        if (inputRef.current.value.toString() !== ' ' && inputRef.current.value.toString().length !== 0)
-            dispatch(addTodo(inputRef.current.value.toString()))
-        else
-            alert('Wrong input')
+        let text = inputRef.current.value.toString().trim()
```

Comment:
store a value in state variable instead of storing it in ref

Reply:
done